dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.httpStat-monitor1.0.jakarta.feature

```diff
@@ -1,10 +1,11 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.httpStat-monitor1.0.jakarta
 Manifest-Version: 1.0
+IBM-API-Package: io.openliberty.http.monitor.mbean; type="ibm-api"
 IBM-Provision-Capability: \
-  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.pages-3.1)(osgi.identity=com.ibm.websphere.appserver.servlet-6.0)(osgi.identity=com.ibm.websphere.appserver.servlet-6.1)))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.pages-3.1)(osgi.identity=com.ibm.websphere.appserver.servlet-6.0)(osgi.identity=io.openliberty.servlet.internal-6.0)(osgi.identity=com.ibm.websphere.appserver.servlet-6.1)(osgi.identity=io.openliberty.servlet.internal-6.1)))", \
```

Comment:
```suggestion
  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.servlet.internal-6.0)(osgi.identity=io.openliberty.servlet.internal-6.1)))", \
```
No need for pages 3.1 since it depends on servlet.  And no need for the public servlet features since they depend on the private servlet feature.